src/Renderer/Shader/TileVS.glsl

```diff
@@ -59,7 +59,8 @@ void main() {
                 }
 
             #elif defined(DATA_TEXTURE_ELEVATION)
-                float   dv  = max(texture2D( dTextures_00[0], vVv ).w, 0.);
+                float   dv  = texture2D( dTextures_00[0], vVv ).w;
+                dv *=zFactor;
```

Comment:
You should be doing `dv *=zFactor;` below (just before `vPosition = vec4( position +  vNormal  * dv ,1.0 )`).
This way it would work with all elevation data sources.

Reply:
i did it before like you said and it work but i dont know why it does not work now if i put it just before vPosition = vec4( position + vNormal * dv ,1.0 )). I think three is a problem with vNormal

Reply:
Hmm.. I don't know what is the underlying issue here, but you need to fix this so the `z factor` feature works independently of the elevation data source.

Reply:
i set zFactor value by default to 1, so it will not impact on elevation data.

Reply:
My point is: we want this to work regardless of the data source used for elevation.
As an example, `zFactor` feature should work on planar.html demo.

Reply:
ok, done